galerautils/src/gu_dbug.c

```diff
@@ -1532,7 +1532,7 @@ GU_DBUGOpenFile(const char *name, int append)
 
     if (name != NULL) {
 	strcpy(_gu_db_stack->name, name);
-	if (strcmp(name, "-") == 0) {
+	if (strlen(name) == 1 && name[0] == '=') {
```

Comment:
Blast, there is indeed a typo, fix coming.
However the change was needed because clang failed to compile it. It (or libc) expanded strcmp() into some HUGE and barely parseable script that resulted in array boundary violation. I knew I needed to comment on that, but commenting code for a code **change** felt strange... The code itself is rather clear though.